src/develop/masks/ellipse.c

```diff
@@ -1893,6 +1911,7 @@ static int _ellipse_get_mask(const dt_iop_module_t *const module, const dt_dev_p
 }
 
 
+#if 0
```

Comment:
to be removed, this is in anyway still in Git history.

Reply:
I hadn't yet decided whether to remove it entirely or move it to math.h in case it proves useful somewhere else to speed up computations involving atan2.  But now that flannelhead has what looks like an even better version, I'll remove it before finalizing the PR.